tests/dag/config/base/bigquery.yml

```diff
@@ -0,0 +1,10 @@
+type: bigquery
```

Comment:
I think we can skip the type  and take always from dbt.yml (you already have target_type)

Reply:
done